React/Base/RCTConvert.m

```diff
@@ -757,7 +757,56 @@ + (UIFont *)UIFont:(UIFont *)font withFamily:(id)family
   BOOL isItalic = NO;
   BOOL isCondensed = NO;
 
+  if ([[self NSString:family] isEqualToString:RCTDefaultFontFamily] ||
+      (!family && !font)) {
+    if (font) {
+      fontSize = font.pointSize ?: RCTDefaultFontSize;
+      fontWeight = RCTWeightOfFont(font);
+      isItalic = RCTFontIsItalic(font);
+    }
+    fontSize = [self CGFloat:size] ?: fontSize;
+    fontWeight = [self RCTFontWeight:weight] ?: fontWeight;
+
+    if (weight) {
+      font = [UIFont systemFontOfSize:fontSize weight:fontWeight];
```

Comment:
This method was only added in iOS 8.2, it will crash on iOS 8.1 and earlier (we need to support 7+)